setup.py

```diff
@@ -36,5 +36,5 @@
         'Topic :: Internet :: WWW/HTTP',
     ],
     test_suite='tests',
-    tests_require=['mock']
+    tests_require=['mock', 'pytest']
```

Comment:
Mhh since we use `tox` to manage testing [and test dependencies](https://github.com/scrapinghub/shub/blob/master/tox.ini#L8), should we just remove this line and the one above? `python setup.py test` fails on my machine.